### ISSUE TITLE
"User specified parameters" should not be dropped in WFS requests

### ISSUE THREAD

**Issue by heidivanparys** [inspire-eu-validation/ets-repository#141]
The description of test step "[Get Capabilities](http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID3eba4a65-41a3-4154-a19b-09c29d4a4a76.html?lang=en#EID4ba669e5-0161-1000-0a4c-4ed9a935a5a9)" says that 

> User specified parameters are dropped in this request.

As far as I can see, the WFS standard actually allows for vendor-specific parameters to be supported. I am aware of the fact that it also says that

> A server shall produce valid results even if the VSPs are missing [...]

![image](https://user-images.githubusercontent.com/11427611/35619073-260611a0-067d-11e8-8ab1-692c88b097d9.png)

The concrete issue we have here, is that we support [user authentication through custom parameters](https://www.kortforsyningen.dk/content/login-on-the-danish-map-supplyen). So our services give an error when these parameters are missing (apart from the Capabilities, those anyone is allowed to see).

Would it be possible to update the test so that

- user specified parameters are not dropped
- alternatively, one can configure whether or not to drop user specified parameters

This issue occurs (at least) in:

- http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID3eba4a65-41a3-4154-a19b-09c29d4a4a76.html?lang=en#EID4ba669e5-0161-1000-0a4c-4ed9a935a5a9
- http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID3eba4a65-41a3-4154-a19b-09c29d4a4a76.html?lang=en#EID4ba67b35-0161-1000-c501-2bfd3f78ad23
- http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID3eba4a65-41a3-4154-a19b-09c29d4a4a76.html?lang=en#EID4ba67bd5-0161-1000-c157-fcb4c185f996
- http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID3eba4a65-41a3-4154-a19b-09c29d4a4a76.html?lang=en#EID4ba67c05-0161-1000-0965-e931906f03cb

**Comment by jonherrmann** [comment]
> As far as I can see, the WFS standard actually allows for vendor-specific parameters to be supported.

That's right and we also use these additional parameters in the [Service is available test case](http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID3eba4a65-41a3-4154-a19b-09c29d4a4a76.html?lang=en#EID4ba6675f-0161-1000-2750-5b251098fca3).

>  I am aware of the fact that it also says that "A server shall produce valid results even if the VSPs are missing"

For all the subsequent test cases, we use the endpoints from the capabilities doc, which is returned in the 'Service is available' test case. A "workaround" for this problem might be to let your service pass the additional parameters from the request to the Capabilities document and add them to all endpoints.

The support for additional parameters (or an additional authentication mechanism) requires changes in the ETS and the validator itself. I will label this issue as a Feature Request.

**Comment by heidivanparys** [comment]
> For all the subsequent test cases, we use the endpoints from the capabilities doc, which is returned in the 'Service is available' test case.

This is not the behaviour I see. The Get Capabilities document that is returned in in the [Service is available test case](http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID3eba4a65-41a3-4154-a19b-09c29d4a4a76.html?lang=en#EID4ba6675f-0161-1000-2750-5b251098fca3) contains a "ticket" parameter.

![image](https://user-images.githubusercontent.com/11427611/35622927-e834e178-0689-11e8-86a2-ba1adf088a7d.png)

The ticket parameter is not included in the requests in the subsequent test cases, e.g. see test case below:

![image](https://user-images.githubusercontent.com/11427611/35623062-5dbeff96-068a-11e8-8211-43846adee66e.png)

**Comment by jonherrmann** [comment]
Ok, I didn't see this. It might be related to issue  #140. I will check this.
